### PR TITLE
Add welcome page and quick replies

### PR DESCRIPTION
**Pull request by geekyme**
Screenshot 2020-11-03 at 12 55 36 AM
Screenshot 2020-11-03 at 12 02 51 PM

### CODE REVIEW

File: bots/faq_bot.py [hunk 1]
```diff
@@ -13,6 +13,9 @@
 SESSION_TIMEOUT_SECONDS = 300
 UNKNOWN_ANSWER = "Sorry, I can't help with that yet. Try to ask another question!"
 UNKNOWN_THRESHOLD = 0.5
+DEFAULT_WELCOME_MESSAGE = "Greetings! I'm the taiwan chat bot! You may ask me anything about taiwan and I'll do my best to answer your questions 🧙 For starters, you may select a question from below 👇"
+WELCOME_QUICK_REPLIES = ["What's gold card?",
+                         "How's the rent?", "Tax in taiwan"]
```
Comment (erickhun) [inline bots/faq_bot.py]: What's gold card? > Gold Card?
Tax in taiwan> Tax in Taiwan?

File: bots/faq_bot.py [hunk 2]
```diff
@@ -13,6 +13,9 @@
 SESSION_TIMEOUT_SECONDS = 300
 UNKNOWN_ANSWER = "Sorry, I can't help with that yet. Try to ask another question!"
 UNKNOWN_THRESHOLD = 0.5
+DEFAULT_WELCOME_MESSAGE = "Greetings! I'm the taiwan chat bot! You may ask me anything about taiwan and I'll do my best to answer your questions 🧙 For starters, you may select a question from below 👇"
```
Comment (erickhun) [inline bots/faq_bot.py]: DEFAULT_WELCOME_MESSAGE = "Greetings! Taiwan chat bot! You may ask me anything about Taiwan and I'll do my best to answer your questions 🧙 For starters, you may select a question from below 👇"

Comment (geekyme) [reply bots/faq_bot.py]: lol this sounds funny -> "Greetings! Taiwan chat bot!" The bot is calling his own name. I think i'll just drop "Taiwan chat bot" here.

**Review by erickhun — APPROVED** [review]
lgtm!

**Pull request merged**